lte/gateway/c/core/oai/tasks/nas/esm/DefaultEpsBearerContextActivation.c

```diff
@@ -415,14 +416,14 @@ status_code_e esm_proc_default_eps_bearer_context_failure(
   *pid = MAX_APN_PER_UE;
 
   if (emm_context) {
-    OAILOG_WARNING(
-        LOG_NAS_ESM,
+    OAILOG_WARNING_UE(
+        LOG_NAS_ESM, emm_context->_imsi64,
         "ESM-PROC  - Default EPS bearer context activation "
         "failure (ue_id=" MME_UE_S1AP_ID_FMT ")\n",
         ue_id);
   } else {
-    OAILOG_WARNING(
-        LOG_NAS_ESM,
+    OAILOG_WARNING_UE(
```

Comment:
`emm_context` does not exist in this else case, let's not change this

Reply:
Resolved

Reply:
Resolved